internal/provider/none/installConfig.go

```diff
@@ -5,10 +5,36 @@ import (
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/host/hostutil"
 	"github.com/openshift/assisted-service/internal/installcfg"
+	"github.com/openshift/assisted-service/internal/network"
 	"github.com/openshift/assisted-service/internal/provider"
 	"github.com/openshift/assisted-service/models"
+	"github.com/samber/lo"
 )
 
+func (p noneProvider) replaceMachineNetworkIfNeeded(cluster *common.Cluster, cfg *installcfg.InstallerConfigBaremetal) {
+	bootstrapHost := common.GetBootstrapHost(cluster)
+	if bootstrapHost != nil {
+		nodeIpAllocations, err := network.GenerateNonePlatformAddressAllocation(cluster, p.Log)
+		if err != nil {
+			p.Log.WithError(err).Warnf("failed to get node ip allocations for cluster %s", *cluster.ID)
+		} else {
```

Comment:
lets return in case of error and remove else, imo will be easier to read

Reply:
Same as here https://github.com/openshift/assisted-service/pull/6257#discussion_r1607998519